### DISCUSSION TITLE
Overlapping Structures in Training Labels

### DISCUSSION THREAD

**Discussion by TimBirtwhistle** [MIC-DKFZ/nnUNet#515] [Q&A] [answered]
Hi,

I'm new to nnUNet but hoping to use it for segementing some organs on CT scans. 

I saw in the documentation that "the label files must contain segmentation maps that contain consecutive integers" My training data set has some overlapping structures so I'd prefer to use the labels 0,1,2,4,8 etc. 

Will this be possible if I specify this in the .json file? If not is there another way to deal with overlapping structures?

Thanks,
Tim

**Comment by FabianIsensee** [comment, marked as answer]
Hi,
you can reuse the code I used to participate in BraTS 2020. This challenge also required me to train with overlapping labels.But this is something that currently needs to be hard coded in the trainer class. Best would be to derive your class from nnUNetTrainerV2BraTSRegions. Labels still need to be consecutive though. 
Best,
Fabian

**Reply by TimBirtwhistle** [reply]
Hi Fabian, thanks for your helpful reply. I'll have a look at the code you suggested and see how i get on.
Best wishes 
Tim
